testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeIgnite.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.env.environment;
+
+import io.trino.testing.TestingProperties;
+import io.trino.tests.product.launcher.docker.DockerFiles;
+import io.trino.tests.product.launcher.docker.DockerFiles.ResourceProvider;
+import io.trino.tests.product.launcher.env.DockerContainer;
+import io.trino.tests.product.launcher.env.Environment;
+import io.trino.tests.product.launcher.env.EnvironmentProvider;
+import io.trino.tests.product.launcher.env.common.StandardMultinode;
+import io.trino.tests.product.launcher.env.common.TestsEnvironment;
+import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
+
+import javax.inject.Inject;
+
+import java.time.Duration;
+
+import static io.trino.tests.product.launcher.docker.ContainerUtil.forSelectedPorts;
+import static io.trino.tests.product.launcher.env.EnvironmentContainers.configureTempto;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+@TestsEnvironment
+public final class EnvMultinodeIgnite
+        extends EnvironmentProvider
+{
+    private static final String IGNITE = "ignite";
+    private static final String IGNITE_NTH = IGNITE + "-";
+    private final ResourceProvider configDir;
+
+    @Inject
+    public EnvMultinodeIgnite(StandardMultinode standardMultinode, DockerFiles dockerFiles)
+    {
+        super(standardMultinode);
+        this.configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment/multinode-ignite/");
+    }
+
+    @Override
+    public void extendEnvironment(Environment.Builder builder)
+    {
+        builder.addContainers(createIgnite(1), createIgnite(2));
+        builder.addConnector("ignite", forHostPath(configDir.getPath("ignite.properties")));
+        configureTempto(builder, configDir);
+    }
+
+    private DockerContainer createIgnite(int number)
+    {
+        String dockerImageName = "ghcr.io/trinodb/testing/ignite:" + TestingProperties.getDockerImagesVersion();
```

Comment:
I believe such an image doesn't exist.

Reply:
I found `<air.test.jvm.additional-arguments>` in `phoenix5.pom`, and try to add the same config in `ignite.pom` found it did not work. wondering how this line works in phoenix?

Reply:
Consult with @wendigo , the Hbase services in Phoenix5 pt is running on JDK11, and will override the jvm.config to support Ignite pt.